content/faq/editing/advanced-map-editing/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: How can I do more advanced map editing?
+description: "Tutorial for editing OpenStreetMap with more advanced tools like ID, Go Map and Vespucci"
+
+taxonomies:
+  faq: ["Map Editing"]
+
+extra:
+  order: 40
+---
+
+Organic Maps includes a simple and easy-to-use editor that you can use to edit the map. The editor is, however, limited and only allows to add simple point features, that means no building outlines, roads, lakes, towns, etc. If you want to change something that can not be edited with the build-in editor, this is the right FAQ page to read.
+
+As all map data used in Organic Maps comes from [OpenStreetMap.org (OSM)](https://www.openstreetmap.org), you can directly update the map there. Your modifications will then be included into Organic Maps with the next map update.
+
+## OpenStreetMap Editors
+
+For editing OSM, there are several options. If you have a desktop PC at hand, it is recommended that you use the ID Editor. The editor is easy to use for beginners, and a bigger screen, mouse, and keyboard make editing easier. The editor runs in your browser, so no installation is required.
+
+If you don't have a desktop PC at hand, you can use the mobile editors Go Map for ios or Vespucci for Android. Go Map is easy to use for beginners, while Vespucci targets more advanced users.
+
+Please pick one of the three editors to proceede.
+
+#### ID Editor
+For editing OpenStreetMap with ID follow these steps:
+
+1. Create a new account or log in at [OpenStreetMap.org](https://www.openstreetmap.org)
+2. Browse to the location you want to edit on OpenStreetMap.org and click *Edit*
+3. *Start the Walkthrough* and follow the short tutorial that explains the ID Editor
+4. Edit the map
+5. Upload your changes
+
+That's it, you are now part of the OSM community. Wait for the next Organic Maps update to see the changes in your favorite map app.
+
+#### Go Map
+For editing OpenStreetMap with Go Map follow these steps:
+
+1. Download the app from the [App Store](https://apps.apple.com/de/app/go-map/id592990211)
+2. Follow the detailed [LearnOSM tutorial](https://learnosm.org/en/mobile-mapping/gomap/)
```

Comment:
Can these tutorial hints be embedded above, where these editors are referenced? Or maybe a separate section "Tutorials" would be better?

Reply:
:+1: I moved the tutorial hints to the editor overview